boot/modeenv.go

```diff
@@ -62,6 +66,10 @@ type Modeenv struct {
 	// asset names to a list of hashes of the asset contents. Used similarly
 	// to CurrentTrustedBootAssets.
 	CurrentTrustedRecoveryBootAssets bootAssetsMap `key:"current_trusted_recovery_boot_assets"`
+	// CurrentKernelCommandLines is stack of the expected kernel command
+	// lines when booting into run mode.
```

Comment:
```suggestion
	// CurrentKernelCommandLines is the stack of the expected kernel command
	// lines when booting into run mode. It will typically only be one element for 
	// normal operations, but may contain two elements during update scenarios.
```
(this is assuming I am correct in the understanding above)